Main: test
CallStack (from HasCallStack):
  error, called at callstack/./Main.hs:4:8 in main:Main
